packages/api/src/external/commonwell/document/document-query.ts

```diff
@@ -112,7 +112,7 @@ export async function queryAndProcessDocuments({
         ignoreDocRefOnFHIRServer,
       });
 
-      reportDocQueryUsage(patient);
+      if (fhirDocRefs.length) reportDocQueryUsage(patient);
```

Comment:
note that ` forceDownload` & ` ignoreDocRefOnFHIRServer` should only be used for internal ops purposes, and are not things we want to bill cxs for - would suggest changing this logic to:

 `if (fhirDocRefs.length &&  forceDownload === undefined &&  ignoreDocRefOnFHIRServer === undefined) reportDocQueryUsage(patient);`